src/scss/includes/utils.scss

```diff
@@ -68,6 +68,10 @@
   font-weight: bold;
 }
 
+.u-mt-8 {
```

Comment:
Not a fan of having utils for both top and bottom. I think we agree in trying to stack elements by using mostly bottom margins, and this would complexify stuff.